### ISSUE TITLE
Compilation for Android fails, can not find libQtAV.so

### ISSUE THREAD

**Issue by javlada** [wang-bin/QtAV#1132]
I updated Qt libraries to 5.11.2 and wanted to compile QtAV for this new version. The android compilation fails with this error:
`del ..\lib_android_\libQtAV.so
move libQtAV.so  ..\lib_android_\ 
process_begin: CreateProcess(NULL, move libQtAV.so "..\lib_android_ ", ...) failed.
make (e=2): System can not find the file.

make[1]: [..\lib_android_\libQtAV.so] Error 2 (ignored)`

I tried to compile it for 5.11.1 and now it fails with the same error too. I think it did work in the past. The file libQtAV.so is created in the /src directory. Is there maybe a change in the directory structure?

Do you have any idea what might be wrong and how to fix it?

I'm using Windows 10, latest QtAV master branch and  Qt 5.11.x.

**Comment by tangjiqiang** [comment]
Did you compile successfully?

**Comment by javlada** [comment]
Not really. I always had to copy the one produced .so file to the destination location and then run the compilation again. The next one was created etc.

So in the end it works but there is something broken in the workflow.
